ReleaseProcessAutomation.IntegrationTests/GitBackedTests.cs

```diff
@@ -32,7 +32,7 @@ public class GitBackedTests
   private string _remotePath;
```

Comment:
Class name: base types should be abstract and end in `Base`. Here, it would be GitBackedTestBase